src/emitters.ts

```diff
@@ -1,11 +1,14 @@
 import * as vscode from "vscode";
 import { KafkaCluster } from "./models/kafkaCluster";
 import { ConnectionId, EnvironmentId } from "./models/resource";
+import { Schema, Subject } from "./models/schema";
 import { SchemaRegistry } from "./models/schemaRegistry";
 
 // NOTE: these are kept at the global level to allow for easy access from any file and track where
 // we .fire() events and where we react to them via .event()
 
+export type EventChangeType = "added" | "deleted";
```

Comment:
Minor: maybe move this down closer to `SubjectChangeEvent`/`SchemaVersionChangeEvent`?